src/datatable/__init__.py

```diff
@@ -85,6 +87,8 @@
     "corr",
     "count",
     "cov",
+    "cummin",
+    "cummax",
```

Comment:
And here.

Reply:
by the way is there any particular reason for the alphabetical arrangement?

Reply:
When we have tons of functions it just makes it easier to navigate and look for some specific things. Both for the source code (developers) and documentation (users).